src/main/java/dev/railroadide/railroad/ui/nodes/ProjectListCell.java

```diff
@@ -31,11 +34,12 @@ public class ProjectListCell extends ListCell<Project> {
     private final RRCard card = new RRCard(14, new Insets(8, 32, 8, 32));
     private final HBox root = new HBox(16);
     private final ImageView icon = new ImageView();
-    private final VBox infoBox = new VBox(7);
+    private final VBox infoBox = new VBox(4);
     private final Label nameLabel = new Label();
     private final Label pathLabel = new Label();
     private final Label lastOpenedLabel = new Label();
     private final RRButton ellipsisButton = new RRButton();
+    private final HBox facetTagsBox = new HBox(5); // 5px spacing between tags
```

Comment:
Unnecessary comment.
```suggestion
    private final HBox facetTagsBox = new HBox(5);
```